week-4/F-object-keys/README.md

```diff
@@ -1,5 +1,18 @@
 ### Goals
 
-- explain `Object.keys()` 
-- good time to mention how `Object` is just another JS object provided by the runtime that we can access anytime (similar to `console`)
-- continue with more advanced exercises that use `Object.keys` but also build up on the previous topics
\ No newline at end of file
+`Object.keys()` allows your to iterate over an object's keys, in the same way that a `for..in` loop would, and return an array of strings of your object's keys.
+
+Object is a type that has built-in methods attached to it - such as `.keys()` or `.values()`. You're all very familiar with `console.log()` - `Object` is helpfully provided to us.
+
+Here is an example output for using `.keys()`:
+
+```js
+var footballClubs = {
+  chelsea_fc: 'England',
+  fc_barcelona: 'Spain',
+  ac_milan: 'Italy'
+};
+
+console.log(Object.keys(footballClubs));
+// returns [ 'chelsea_fc', 'fc_barcelona', 'ac_milan' ]

```

Comment:
I wouldn't say "returns" here to avoid confusion with returning values from functions - just say 'prints "....."'. Logging is a side-effect rather than a return value.